content/languages/java/junit.md

```diff
@@ -2,3 +2,61 @@
 title: JUnit
 kind: reference
 ---
+
+You can use JUnit [4](https://junit.org/junit4/) or [5](https://junit.org/junit5/) in Codewars. 5 offers nesting, pretty name display and multiple test classes.
+
+Note that JUnit assertions use `(expected, actual)` parameter ordering rather than the typical `(actual, expected)`.
```

Comment:
This statement is biased :) After years of work with JUnit and NUnit I was surprised that anything uses `(actual, expected)`

Reply:
OK now I double checked and I see that NUnit uses `(actual, expected)` too.
It's actually interesting how I manage to switch between JUnit and NUnit and get the order of arguments right, but I can never remember it for Chai or Python framework.

Reply:
Yeah, it's a bit biased. I think JUnit and PHPUnit are the "only" (I'm sure there are others that I'm not familiar with) that use (exp, act), off the top of my head. We can still make the statement more neutral, but I tend to see more issues with PHP having asserts in the wrong direction than other languages--again, totally anecdotal. Feel free to PR a fix.